.github/workflows/stale-needs-repro.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-label: "stale"
+          any-of-issue-labels: "needs-repro"
+          close-issue-reason: "not_planned"
+          ignore-issue-updates: true
+          stale-issue-message: |
+            Thank you for using Quarto and reporting an issue!
+            
+            Unfortunately, this issue is now considered stale because it has been open 30 days without providing a "working" reproducible example.
```

Comment:
```suggestion
            Unfortunately, this issue is now considered stale because it has been opened since 14 days without providing a "working" reproducible example to help us investigate.
```